framework/src/Volo.Abp.Uow/Volo/Abp/Uow/UnitOfWorkExtensions.cs

```diff
@@ -25,7 +25,7 @@ public static void AddItem<TValue>([NotNull] this IUnitOfWork unitOfWork, string
         }
```

Comment:
We may always use `unitOfWork.Items[key] = value;` without the `if` statement. I don't know why it was designed like that (checking containskey), however, if we consider concurrency now, this code is being a problem:

````csharp
if (!unitOfWork.Items.ContainsKey(key))
        {
            unitOfWork.Items[key] = value;
        }
````

Because another thread may change the `Items` after `ContainsKey`, but before setting the value.

Reply:
OK, I undid the changes and removed the if statement, it should fix the problem